src/main/java/ja/workshops/spring/części/konfiguracja/programistyczna/c_specjalistyczne_ziarna/RozsądneRepozytorium.java

```diff
@@ -0,0 +1,60 @@
+package ja.workshops.spring.części.konfiguracja.programistyczna.c_specjalistyczne_ziarna;
+
+import org.springframework.stereotype.Repository;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Adnotacja {@link org.springframework.stereotype.Repository @Repository} informuje nas, że mamy do czynienia z "Repozytorium" (obviously) - obiektem
+ * udostępniającym API do manipulacji danymi, a więc abstrakcją zbioru obiektów.
+ * <br/>
+ * W tym momencie przejdź proszę do dokumentacji adnotacji {@link org.springframework.stereotype.Repository @Repository} i przeczytaj ją całą.
+ * <br/>
+ * Repozytoriów najczęściej używamy w powiązaniu z bazami danych, jednak w tym
+ * przykładzie (w imię prostoty), skorzystamy z kolekcji obiektów w pamięci.
+ * Dodatkowo, zgodnie z dokumentacją repozytorium powinno wspierać przynajmniej 3 podstawowe operacje
+ *  <ul>
+ *      <li>Zapis</li>
+ *      <li>Odczyt</li>
+ *      <li>Wyszukiwanie</li>
+ *  </ul>
+ * ale w imię prostoty (znowu) ograniczymy te funkcjonalności do wyłącznie odczytu.
```

Comment:
"ale (znowu) w imię prostoty" albo "ale w imię (znowu) prostoty", inaczej podkreślasz, że znowu ograniczymy. Chyba, że to było celem.